queue_services/entity-auth/k8s/knative/dc.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: $NAME
+  generation: $GENERATION
+  labels:
+    cloud.googleapis.com/location: northamerica-northeast1
+spec:
+  template:
+    metadata:
+      labels:
+        run.googleapis.com/startupProbeType: Default
+      annotations:
+        run.googleapis.com/client-name: cloud-console
+        run.googleapis.com/cloudsql-instances: $CLOUDSQL_INSTANCES
+        autoscaling.knative.dev/maxScale: $MAXSCALE
+    spec:
+      containerConcurrency: $CONTAINER_CONCURRENCY
+      timeoutSeconds: $TIMEOUT_SECONDS
+      serviceAccountName: $SERVICE_ACCOUNTNAME
+      containers:
+      - image: >-
+          $CONTAINER_IMAGE
+        ports:
+        - name: http1
+          containerPort: $CONTAINER_PORT
+        env:
+        - name: DATABASE_HOST
+          value: $DATABASE_HOST
+        - name: DATABASE_NAME
+          value: $DATABASE_NAME
+        - name: DATABASE_PASSWORD
+          value: $DATABASE_PASSWORD
+        - name: DATABASE_PORT
+          value: $DATABASE_PORT
+        - name: GUNICORN_PROCESSES
+          value: $GUNICORN_PROCESSES
+        - name: GUNICORN_THREADS
+          value: GUNICORN_THREADS
+        - name: DATABASE_UNIX_SOCKET
+          value: $DATABASE_UNIX_SOCKET
+        - name: DATABASE_USERNAME
+          value: DATABASE_USERNAME
+        - name: AUDIENCE
+          value: $AUDIENCE
+        - name: PUBLISHER_AUDIENCE
+          value: $PUBLISHER_AUDIENCE
+        - name: AUTH_SVC_URL
+          value: $AUTH_SVC_URL
+        - name: ACCOUNT_SVC_AUTH_URL
+          value: $ACCOUNT_SVC_AUTH_URL
+        - name: ACCOUNT_SVC_CLIENT_ID
+          value: $ACCOUNT_SVC_CLIENT_ID
+        - name: ACCOUNT_SVC_CLIENT_SECRET
+          value: $ACCOUNT_SVC_CLIENT_SECRET
```

Comment:
These are the config we need to retrieve from `entity-auth`

Reply:
we had these values in vault.
![image](https://github.com/bcgov/lear/assets/44277752/10371086-c89b-4268-af45-21d42053beed)

![image](https://github.com/bcgov/lear/assets/44277752/0423186a-13f7-4369-a302-0559035b5045)